src/meta-srv/src/service/store/etcd.rs

```diff
@@ -98,6 +98,40 @@ impl KvStore for EtcdStore {
         Ok(PutResponse { header, prev_kv })
     }
 
+    async fn batch_get(&self, req: BatchGetRequest) -> Result<BatchGetResponse> {
```

Comment:
We need to add a unit test for `batch_get to` [meta client](https://github.com/GreptimeTeam/greptimedb/blob/develop/src/meta-client/src/client.rs#L736) and we need to Ensure that any behavior of the memory store and etcd store is consistent

Reply:
I will add unit test soon.